packages/integrations/vercel/README.md

```diff
@@ -87,6 +87,26 @@ vercel deploy --prebuilt
 
 To configure this adapter, pass an object to the `vercel()` function call in `astro.config.mjs`:
 
+### analytics
+
+> **Type:** `boolean`
+> **Available for:** Serverless, Edge, Static
+
+Use this property to enable Vercel Analytics (including Web Vitals and Audiences).
```

Comment:
Suggestion to link to Vercel’s docs for more information, be slightly more explicit on how to enable this feature, and give some more context about what this does.

```suggestion
You can enable [Vercel Analytics](https://vercel.com/analytics) (including Web Vitals and Audiences) by setting `analytics: true`. This will inject Vercel’s tracking scripts into all your pages.
```